internal/testutil/acc/shared_resource.go

```diff
@@ -32,7 +34,16 @@ func cleanupSharedResources() {
 		fmt.Printf("Deleting execution cluster: %s, project id: %s\n", sharedInfo.clusterName, projectID)
 		deleteCluster(projectID, sharedInfo.clusterName)
 	}
-
+	if !ExistingStreamInstanceUsed() && sharedInfo.streamInstanceName != "" {
```

Comment:
```suggestion
	if sharedInfo.streamInstanceName != "" {
```

I think sharedInfo.streamInstanceName is not updated if ExistingStreamInstanceUsed is true, so just need to check  sharedInfo.streamInstanceName != ""

Reply:
Thank you. Changed